app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -369,6 +370,16 @@ class StateFragmentTest {
     Intents.release()
   }
 
+  @Test
+  fun testContentCard_forDemoExploration_withCustomOppiaTags_displaysParsedHtml() {
+    ActivityScenario.launch(ContentCardTestActivity::class.java).use {
+      val htmlResult =
+        "Hi, welcome to Oppia! is a tool that helps you create interactive learning activities that can be continually improved over time.\n\n" +
+            "Incidentally, do you know where the name 'Oppia' comes from?\n\n"
+      onView(atPosition(R.id.state_recycler_view, 0)).check(matches(hasDescendant(withText(htmlResult))))
```

Comment:
Change to `withText(containsString(htmlResult)))))` that way you won't need to check extra `\n` and all.

Reply:
done.